electron-src/process/js/scheduletable.js

```diff
@@ -35,57 +16,92 @@ class ScheduleTable extends React.Component {
+                containerDataList.push(
+                    {
+                        id: index,
+                        tutorName: tutor,
+                        tutorCourse: assignments[row].courses[0].name
+                    });
+                index++;
             }
-            j++;
+            console.log(containerDataList);
+            containerList.push(<div className="col-2">
+                                    <div className="row">
+                                        <div className="col text-center"
+                                             style={{background: "#5bc0de", color: "white", padding:12}}>
+                                            {scheduleShifts[col].shift.day}
+                                        </div>
+                                    </div>
+                                    <div className="row">
+                                        <div className="col">
+                                            <Container id={col + 1} list={containerDataList} />
+                                        </div>
+                                    </div>
+                                </div>);
         }
+
+        const listOne = [
+            {id: 1, tutorName: "Lily", tutorCourse: "CS 200"},
+            {id: 2, tutorName: "Sally", tutorCourse: "CS 540"},
+            {id: 3, tutorName: "Wid", tutorCourse: "CS 800"}
+        ];
+
+        const listTwo = [
+            {id: 4, tutorName: "fan", tutorCourse: "CS 1110"},
+            {id: 5, tutorName: "table", tutorCourse: "CS 524"},
+            {id: 6, tutorName: "chair", tutorCourse: "CS 10"}
+        ];
+
+        const listThree = [
+            {id: 7, tutorName: "cat", tutorCourse: "CS 885"},
+            {id: 8, tutorName: "dog", tutorCourse: "CS 243"},
+            {id: 9, tutorName: "pangolin", tutorCourse: "CS 789"}
+        ];
+
         return (
-            <div className="card" style={{overflow: "auto", background: "#049cdb", border: "#333", color: "#ffffff"}}>
-                <div className="card-block">
-                    <h3 className="card-header">
-                        Schedule: Schedule 1
-                    </h3>
-                    <div className = "card-body" style={{color: "#000000"}}>
-                    <table className="table">
-                        <thead>
-                            <tr>
-                                {a}
-                            </tr>
-                        </thead>
-                        <tbody>
-                            {b}
-                        </tbody>
-                    </table>
-                    </div> {/*tool bar */}
+            <div className="container">
+                <div className="row">
+                    <div className="col">
+                        <div className="row" style={{background: "#5bc0de", color: "white", padding: 15}}>
+                            <h3>Schedule: Schedule 1</h3>
```

Comment:
This should be dynamic to allow for changing titles depending on which schedule is selected. Not a huge deal for now, but we want this to be dynamic. Not sure if this is the appropriate place to comment this, but I'll leave it here anyway.